pandas/core/strings.py

```diff
@@ -2466,7 +2474,27 @@ def cat(self, others=None, sep=None, na_rep=None, join="left"):
         2    -c
         dtype: object
 
-        For more examples, see :ref:`here <text.concatenate>`.
+        When `others` is a Series/Index/DataFrame, it is advisable
+        to use `to_numpy()` while passing it to the function to
+        avoid unexpected results.
+
+        The following example demostrates this:
+
+        If `others` is a Series/Index/DataFrame and `to_numpy()` is
+        **not** used:
+
+        >>> idx = pd.Index(["a", "b", "c", "d", "e"])
+        >>> ser = pd.Series(["f", "g", "h", "i", "j"])
+        >>> idx.str.cat(ser) # the caller is an Index here
+        Index([nan, nan, nan, nan, nan], dtype='object')
+
+        When `to_numpy()` **is used**:
```

Comment:
Could it be

```
**is not** used
```
above, and
```
**is** used
```
here, just for consistency?

Reply:
Sure, can agree with that.